provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -269,6 +269,21 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 					listplanmodifier.RequiresReplace(),
 				},
 			},
+			"aws_additional_compute_security_group_ids": schema.ListAttribute{
+				Description: "AWS additional compute security group ids.",
+				ElementType: types.StringType,
+				Optional:    true,
```

Comment:
What about updating those attributes? we would like to support it? if no please add a plenmodifier 
If we would like to support updating it please add the logic in update function

Reply:
There's no current functionality to update the attributes. It will be supported at some point in the future

Reply:
So you should block it with plan modifier 
```
PlanModifiers: []planmodifier.String{
					common.Immutable(),
```